apps/page/option/banned_word.ts

```diff
@@ -28,25 +28,25 @@ export function toBannedTarget(value: string): BannedTarget {
 }
 
 export default class BannedWords {
-  private addButton: HTMLInputElement;
+  // private addButton: HTMLInputElement;
+  private addButton;
 
-  private addText: HTMLInputElement;
+  private addText;
 
-  private wordList: HTMLDivElement;
+  private wordList;
 
   constructor() {
-    this.addButton = document.getElementById(
-      "bannedWordAddButton",
-    ) as HTMLInputElement;
-    this.addText = document.getElementById(
-      "bannedWordAddText",
-    ) as HTMLInputElement;
-    this.wordList = document.getElementById("bannedWordList") as HTMLDivElement;
-
-    this.addButton.addEventListener("click", async () => {
+    this.addButton = document.getElementById("bannedWordAddButton");
+    this.addText = document.getElementById("bannedWordAddText");
+    this.wordList = document.getElementById("bannedWordList");
+
+    this.addButton?.addEventListener("click", async () => {
+      if (!(this.addText instanceof HTMLInputElement)) {
+        throw new Error("bannedWordAddText is not HTMLInputElement");
+      }
       const word = this.addText.value;
       if (word === "") {
-        return;
+        throw new Error("bannedWordAddText is not HTMLInputElement");
```

Comment:
The "word" here refers to a string input by the user, so it's simply being ignored.
Please revert the change.